provider/database.py

```diff
@@ -18,7 +18,8 @@
 from cloudant.result import Result
 
 class Database:
-    dbname = "ow_kafka_triggers"
+    db_prefix = os.environ['DB_PREFIX']
+    dbname = db_prefix + 'ow_kafka_triggers'
```

Comment:
I see. I think you are essentially correct, but it doesn't seem very likely to me. Of all the things you might want to parameterize about this service, the DB name seems rather internal, and unimportant. We can always handle this if/when someone specifically has a need for it. They can already set the `db_prefix` to ensure they end up with a unique name.